provider/resources.go

```diff
@@ -1127,16 +1117,6 @@ func ProviderFromMeta(metaInfo *tfbridge.MetadataInfo) *tfbridge.ProviderInfo {
 			"aws_batch_job_definition":      {Tok: awsResource(batchMod, "JobDefinition")},
 			"aws_batch_job_queue":           {Tok: awsResource(batchMod, "JobQueue")},
 			"aws_batch_scheduling_policy":   {Tok: awsResource(batchMod, "SchedulingPolicy")},
-			// Budgets
-			"aws_budgets_budget": {
-				Tok: awsResource(budgetsMod, "Budget"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"cost_filters": {
-						Name:               "costFilterLegacy",
-						DeprecationMessage: "The now-deprecated original cost filters. Use CostFilters instead.",
-					},
-				},
-			},
```

Comment:
Is this a no-op?

Reply:
No. This is an outdated mapping for `cost_filters` which [does not exist upstream](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/budgets_budget) - note that `cost_filter`, singular, does very much exist. I removed the resource entirely to be picked up by auto resourcing.

Reply:
update: This field was removed on [upstream v5.0.0](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.0.0)